spock-spring/src/test/groovy/org/spockframework/spring/docs/DetachedJavaConfig.groovy

```diff
@@ -0,0 +1,25 @@
+package org.spockframework.spring.docs
+
+import org.springframework.context.annotation.Bean
+import spock.mock.DetachedMockFactory
+
+//tag::javaconfig[]
+public class DetachedJavaConfig {
+  def mockFactory = new DetachedMockFactory()
+
+  @Bean
+  GreeterService serviceMock() {
+    return mockFactory.Mock(GreeterService)
+  }
```

Comment:
So this example will add a `GreeterService` `Bean` to the Spring context?

Since I did not knew this before, I'd love to see this solution mentioned in the docs as well.